bin/deploy.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+PUBLISH=0
+CHEF_VER="latest"
```

Comment:
Where do we actually set the default chef version? Is it here or elsewhere in the process?